packages/uswds-core/src/styles/placeholders/_forms.scss

```
@@ -25,4 +26,8 @@ $input-select-margin-right: 1.5;
   max-width: units($theme-input-max-width);
   padding: units(1);
   width: 100%;
+
+  @media (forced-colors: active) {
+    border: $border-high-contrast;
+  }
```

Couldn't we re-use the high contrast border mixin here?

```suggestion
  @include u-disabled-high-contrast-border;
```

---

Resolved in [78b83e7](https://github.com/uswds/uswds/pull/5295/commits/78b83e70bbd2b739c5806d0920a283dc584b8568)